photoCollection/photoCollection/PhotoDetailViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PhotoDetailViewController.swift
+//  photoCollection
+//
+//  Created by beth on 1/30/20.
+//  Copyright © 2020 elizabeth wingate. All rights reserved.
+//
+
+import UIKit
+
+//part4 step 1
+class PhotoDetailViewController: UIViewController {
+    
+    //part4 step 2
+    var photoController: PhotoController?
+    var photo: Photo?
+    var themeHelper: ThemeHelper?
+    
+    @IBOutlet weak var addPhotoImageView: UIImageView!
+    @IBOutlet weak var addTitleTextField: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateViews()
+
+        // Do any additional setup after loading the view.
```

Comment:
Before you commit and push your code make sure to delete extra whitespace and commented out code.